pm_dftb_opt_write/pm_dftb_opt_write.xml

```diff
@@ -1,4 +1,5 @@
-<tool id="pm_dftb_opt_write" name="Generate DFTB+ outputs (dftb+)" version="0.1.0" python_template_version="3.5" profile="22.01">
+<tool id="pm_dftb_opt_write" name="DFTB+ write" version="0.1.0" python_template_version="3.5" profile="22.01">
```

Comment:
I'd be inclined to drop the `_write` from this and the UEP tool ids, they're slightly confusing IMO. `pm_muairss_write` is different as `pm-muairss` has 'write' and 'read' modes, whereas here the 'opt' is short for 'optimise'